charts/images.yaml

```diff
@@ -151,7 +151,7 @@ images:
 - name: csi-driver
   sourceRepository: github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver
   repository: registry.k8s.io/cloud-provider-gcp/gcp-compute-persistent-disk-csi-driver
-  tag: "v1.8.1"
+  tag: "v1.7.2"
```

Comment:
Wasn't it `1.7.3` previously?
https://github.com/gardener/gardener-extension-provider-gcp/pull/503/files